ethportal-peertest/src/scenarios/find.rs

```diff
@@ -0,0 +1,84 @@
+use serde_json::{json, Value};
+
+use crate::{
+    jsonrpc::{make_ipc_request, JsonRpcRequest, HISTORY_CONTENT_VALUE},
+    Peertest, PeertestConfig,
+};
+use trin_core::jsonrpc::types::Params;
+
+pub fn test_trace_recursive_find_content(_peertest_config: PeertestConfig, peertest: &Peertest) {
+    let uniq_content_key = "0x0015b11b918355b1ef9c5db810302ebad0bf2544255b530cdce90674d5887bb286";
+    // Store content to offer in the testnode db
+    let store_request = JsonRpcRequest {
+        method: "portal_historyStore".to_string(),
+        id: 11,
+        params: Params::Array(vec![
+            Value::String(uniq_content_key.to_string()),
+            Value::String(HISTORY_CONTENT_VALUE.to_string()),
+        ]),
+    };
+
+    let store_result = make_ipc_request(&peertest.bootnode.web3_ipc_path, &store_request).unwrap();
+    assert_eq!(store_result.as_str().unwrap(), "true");
+
+    // Send trace recursive find content request
+    let request = JsonRpcRequest {
+        method: "portal_historyTraceRecursiveFindContent".to_string(),
+        id: 12,
+        params: Params::Array(vec![json!(uniq_content_key)]),
+    };
+
+    let result = make_ipc_request(&peertest.nodes[0].web3_ipc_path, &request).unwrap();
+    assert_eq!(result["content"], json!(HISTORY_CONTENT_VALUE.to_string()));
+    assert_eq!(
+        result["route"],
+        json!([{"enr": &peertest.bootnode.enr.to_base64(), "distance": 256}])
+    );
+}
+
+// This test ensures that the jsonrpc channels don't close if there is an invalid RFC request
```

Comment:
```suggestion
// This test ensures that the jsonrpc channels don't close if there is an invalid RPC request
```

Reply:
`RFC` = "Recursive Find Content" - since this test will only cover cases with invalid params for a recursive find content request, not all rpc request. But I see the point that this is confusing, so I just spelled it out